stingray/lightcurve.py

```diff
@@ -1361,16 +1485,18 @@ def _apply_gtis(self):
         """
         check_gtis(self.gti)
 
-        good = create_gti_mask(self.time, self.gti, dt=self.dt)
+        good = self.mask
```

Comment:
Where does `self.mask` get created again? Is that one of the properties?

Reply:
Yes.